Packs/CimTrak-SystemIntegrityAssurance/Integrations/CimTrak/CimTrak.yml

```diff
@@ -0,0 +1,2334 @@
+    - contextPath: CimTrak.Object.logsByDays
+      description: Logs By Days
+      type: boolean
+    - contextPath: CimTrak.Object.requireNotes
+      description: Require Notes
+      type: boolean
+    - contextPath: CimTrak.Object.inService
+      description: In Service
+      type: string
+    - contextPath: CimTrak.Object.children
+      description: Children
+      type: number
+    - contextPath: CimTrak.Object.events
+      description: Events
+      type: number
+    - contextPath: CimTrak.Object.intrusions
+      description: Intrusions
+      type: number
+    - contextPath: CimTrak.Object.intrusionSize
+      description: Intrusion Size
+      type: number
+    - contextPath: CimTrak.Object.objectId
+      description: Object Id
+      type: number
+    - contextPath: CimTrak.Object.objectStatus
+      description: Object Status
+      type: number
+    - contextPath: CimTrak.Object.objectSubType
+      description: Object SubType
+      type: number
+    - contextPath: CimTrak.Object.objectType
+      description: Object Type
+      type: number
+    - contextPath: CimTrak.Object.parentId
+      description: Parent Id
+      type: number
+    - contextPath: CimTrak.Object.revisions
+      description: Revisions
+      type: number
+    - contextPath: CimTrak.Object.templateId
+      description: Template Id
+      type: number
+    - contextPath: CimTrak.Object.securityAdd
+      description: Security Add
+      type: boolean
+    - contextPath: CimTrak.Object.securityEdit
+      description: Security Edit
+      type: boolean
+    - contextPath: CimTrak.Object.securityLock
+      description: Security Lock
+      type: boolean
+    - contextPath: CimTrak.Object.securityReport
+      description: Security Report
+      type: boolean
+    - contextPath: CimTrak.Object.securityUnlock
+      description: Security Unlock
+      type: boolean
+    - contextPath: CimTrak.Object.securityView
+      description: Security View
+      type: boolean
+    - contextPath: CimTrak.Object.warnMinutes
+      description: Warn Minutes
+      type: number
+    - contextPath: CimTrak.Object.contact
+      description: Contact
+      type: string
+    - contextPath: CimTrak.Object.createDate
+      description: Create Date
+      type: string
+    - contextPath: CimTrak.Object.description
+      description: Description
+      type: string
+    - contextPath: CimTrak.Object.location
+      description: Location
+      type: string
+    - contextPath: CimTrak.Object.name
+      description: Name
+      type: string
+    - contextPath: CimTrak.Object.objectPath
+      description: Object Path
+      type: string
+    - contextPath: CimTrak.Object.url
+      description: URL
+      type: string
+    - contextPath: CimTrak.Object.agentObjectId
+      description: Agent Object Id
+      type: number
+  dockerimage: demisto/python3:3.10.1.25933
+  isfetch: true
+  longRunning: true
+  runonce: false
+  script: ''
+  subtype: python3
+  type: python
```

Comment:
@kigerjoel You might want to add the "defaultclassifier" key so it would be automatically assign the relevant cllasifier.